src/main/java/Bunbun/Bunbun.java

```diff
@@ -0,0 +1,61 @@
+package Bunbun;
+
+import Bunbun.utils.Command;
+import Bunbun.utils.Parser;
+import Bunbun.utils.Storage;
+import Bunbun.utils.TaskList;
+import Bunbun.utils.Ui;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * This class implements a chatbot by the name of Bunbun.
+ *
+ * @author Eda Yeo
+ * @version CS2103T AY24/25 Semester 1
+ */
+public class Bunbun {
+
+    private Storage storage;
+    private TaskList list;
+    private Ui ui;
+
+    public Bunbun(String filePath) {
+        this.storage = new Storage(filePath);
+        this.ui = new Ui();
+        this.list = new TaskList(this.storage, this.ui);
+    }
+
+    public void run() {
+        this.ui.startScreen();
+        storage.initializeTaskFile();
```

Comment:
The variables this.storage, this.ui, and this.list are consistently using the this. keyword for instance variables, which is a good practice. To maintain consistency, ensure that all instance variables throughout the code follow this convention.